### PR TITLE
Only show controls on Mac

### PR DESCRIPTION
**Pull request by paulrouget**
Closes: #915 #925 #913

### CODE REVIEW

File: css/theme.css [hunk 1]
```diff
@@ -42,6 +42,10 @@ body {
   border-radius: 6px;
 }
 
+body.use-native-titlebar {
+  border-radius: 0;
+}
```
Comment (Gozala) [inline css/theme.css]: @paulrouget Can you write a comment explaining the story about the border-radius here ? I have no idea why it has being there in first place or why is it that we don't want it if native titlebar is used.

File: src/browser/index.js [hunk 2]
```diff
@@ -39,6 +40,9 @@ if (isReload) {
   window.application.address(UI.LiveReload);
 }
 
+document.body.classList.toggle('use-native-titlebar',
+                               Runtime.useNativeTitlebar());
+
```
Comment (Gozala) [inline src/browser/index.js]: I wonder if we should just add `OS.platform()` to the `ClassList` instead.

Comment (paulrouget) [inline src/browser/index.js]: I'd prefer to keep the logic that tells if native titlebar is used or not, isolated.

Eventually, the runtime will tell us what kind of titlebar is used, and we will just need to change what's within runtime.js.

File: css/theme.css (hunk 1 above)
Comment (paulrouget) [inline css/theme.css]: ok

**Comment by paulrouget** [comment]
Added a comment.

**Pull request merged**